terraform/datadog/router.tf

```diff
@@ -66,6 +66,22 @@ resource "datadog_monitor" "route_update_latency" {
   }
 }
 
+# This variable is used in the resource with same name to decide in which
+# environments it should be enabled or disable.
+#
+# The resource will refer this variable using a `lookup` function in the count
+# attribute.
+#
+# To enable the resource for a given environment, add an entry here.
+variable "__datadog_monitor_total__routes_drop__enabled" {
+  description = "Selector to enable/disable the named resource"
+
+  default = {
+    "dev"  = 1
+    "prod" = 1
```

Comment:
It is really strange, I was pretty sure I changed this. I don't know what happened here. I will change it again.

I will simplify to only use a simple variable with tfvars, and explicitly disable it for CI and staging with default enabled.  

Not sure what you mean with "That would match the other flags we use."